v22.1/aws-dms.md

```diff
@@ -5,27 +5,33 @@ toc: true
 docs_area: develop
```

Comment:
Should we try and keep the file name the same / closer to the title of the page? Unless there is a reason not to here that I don't know..